site/index.ejs

```diff
@@ -42,7 +42,7 @@ m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 ga('create', 'UA-16654919-1', 'auto');
 ga('send', 'pageview');
     </script><% } %>
-    <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7838134/7848772/css/fonts.css">
+    <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7838134/6842972/css/fonts.css" />
```

Comment:
Not yet. This is introducing a new font pack, which is breaking in Safari (see this, @samsmith1983), and we will have to repack things again before going to live. @asavin understands these details better than me to be able to explain.